src/sut/dumblog/src/Dumblog/Journal/Worker.hs

```diff
@@ -81,7 +83,8 @@ worker journal metrics wi = go (wiEvents wi)
           Metrics.measure metrics Latency latency
           Metrics.measure metrics (case input of
                                      ClientRequest (Write {}) -> ServiceTimeWrites
-                                     ClientRequest (Read {})  -> ServiceTimeReads) serviceTime
+                                     ClientRequest (Read {})  -> ServiceTimeReads
+                                     _othrwise -> error "impossible") serviceTime
```

Comment:
This is only `impossible` if we haven't added a backup peer right

Reply:
Hmm, not sure what you mean. The reason this is impossible is because if the `output` is a `ClientResponse` (which we pattern matched on further up) then the `input` must be a `ClientRequest`.

Reply:
If that's not what you meant, let me know and I'll fix it in the next PR.

Reply:
Well not necessary right, `Ack` also gives a `ClientResponse` even though it is an `InternalMessageIn`